src/Api/IriConverterInterface.php

```diff
@@ -43,7 +43,7 @@ public function getItemFromIri(string $iri, array $context = []);
      * @throws InvalidArgumentException
      * @throws RuntimeException
      */
-    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string;
+    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH, $context = []): string;
```

Comment:
We can't change this interface it breaks the backward compatibility :/.

Reply:
hi, there is a default value,  it should not break the compatibility, can you explain it further ?

Reply:
Sure, we respect the symfony backward compatibility promise and we can't add arguments:

https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces (see the `Add argument with a default value => No`.

Don't worry about this yet, I'm sure there's a solution that doesn't involve adding `context` here!

Reply:
A solution is to add a new interface extending the existing one. See https://github.com/api-platform/core/blob/master/src/DataProvider/ContextAwareCollectionDataProviderInterface.php#L32 for instance.